suites/pacific/rgw/tier-2_rgw_ssl_s3tests.yaml

```diff
@@ -165,3 +165,45 @@ tests:
       module: test_s3.py
       name: execute s3tests
       polarion-id: CEPH-83575225
+
+  - test:
+      name: Deploy RGW Ingress daemon
+      abort-on-fail: true
+      desc: Add RGW and HA Proxy services using spec file.
+      module: test_cephadm.py
+      polarion-id: CEPH-83574607
+      config:
+        steps:
+          - config:
+              command: apply_spec
+              service: orch
+              specs:
+                - service_type: rgw
+                  service_id: my-rgw
+                  placement:
+                    nodes:
+                      - node4
+                      - node5
+                  spec:
+                    rgw_frontend_port: 8080
+          - config:
+              command: shell
+              args: # sleep to get all services deployed
+                - sleep
+                - "60"
+          - config:
+              command: apply_spec
+              service: orch
+              specs:
+                - service_type: ingress
+                  service_id: rgw.my-rgw
+                  placement:
+                    nodes:
+                      - node4
+                      - node5
+                  spec:
+                    backend_service: rgw.my-rgw
+                    virtual_ip: 10.0.208.0/22
```

Comment:
can we look for floating IPS from OSP?

Reply:
The intent is not to hardcode, ultimately we need to establish a way to get floating IPs from OSP which require a framework change. I can remove this line ,since we are creating Virtual IPs in the helper.py. This PR is just to check if the daemons are coming up with SSL . Can we create a Issue in CephCI for the floating IPs?

Reply:
Sure, let's create a Github ticket.

Reply:
I will create a ticket for this.